framework/src/modules/chain/transport/transport.js

```diff
@@ -87,20 +85,9 @@ class Transport {
 	 * @emits signature/change
 	 * @todo Add description for the params
 	 */
-	// eslint-disable-next-line class-methods-use-this
-	handleBroadcastSignature(signature, broadcast) {
-		if (broadcast) {
```

Comment:
You've removed this check which likely is still needed.

Reply:
This was not used in anywhere it was always called with true

Reply:
If broadcast is disabled via config file, you're gonna fill the queues with transactions which won't be released. We were preventing this with that check.

Reply:
https://github.com/LiskHQ/lisk-sdk/pull/4479/files#diff-eafdfefa460d6943a1a938936ca754b5L595
That's a good point, but actually it was not been handled in that way before.

Reply:
A good point forgotten because before wasn't handled?

Reply:
yes before it wasn't handled, and i think no-broadcast was just for testing if i remember correctly.
I will create an issue to remove that